app/controllers/users_controller.rb

```diff
@@ -1,6 +1,14 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, except: %i(show new create)
```

Comment:
chị nghĩ show thì cũng phải check logged_in chứ em
```suggestion
  before_action :logged_in_user, except: %i(new create)
```